client/src/includes/tabs.js

```diff
@@ -152,13 +153,6 @@ class Tabs {
       tabContent.hidden = false;
     }
 
-    if (this.state.initialPageLoad) {
-      // On first load set the scroll to top to avoid scrolling to active section and header covering up tabs
-      setTimeout(() => {
-        window.scrollTo(0, 0);
-      }, this.state.transition * 2);
-    }
-
```

Comment:
As we discussed today, this doesn't do anything and may actually interfere with our scroll to anchor functionality. See commit message for details.